### PR TITLE
update for img*.xyz sites e.g. imgbigo.xyz

### PR DESCRIPTION
**Pull request by Yuki2718**
`http://imgbigo.xyz/aucw5bhn`

Example of a regular expression -> https://regex101.com/r/eskB1S/1

### CODE REVIEW

File: EnglishFilter/sections/general_url.txt [hunk 1]
```diff
@@ -12,6 +12,7 @@
 .com/pw/waWQiOjEw
 ! https://github.com/AdguardTeam/AdguardFilters/pull/83787
 ! ex. http://plqbxvnjxq92.com/gs6h79qcg?key=693ca25a821bb830696c0bf0ccd7c072
+/^http:\/\/img[a-z]+\.xyz(\/[a-z]{2,4}){2}\.js$/$script,~third-party
```
Comment (zloyden) [inline EnglishFilter/sections/general_url.txt:15]: As far as I can tell after checking, this seems normal, but please provide a list of examples from https://regex101.com/

Comment (Yuki2718) [reply EnglishFilter/sections/general_url.txt:15]: Oh, sorry I forgot. Is adding `ex. http://imgbigo.xyz/bb/cc.js` line enough?

**Comment by zloyden** [comment]
@Yuki2718 what about the link from `https://regex101.com/`?

**Comment by Yuki2718** [comment]
@zloyden You saw my comment about `https://regex101.com/r/eskB1S/1`? I don't know much about regex101.

**Comment by zloyden** [comment]
No, I didn't see it. It's ok for now, but add some more examples.
For instance, https://regex101.com/r/jLYcKf/1

**Comment by Yuki2718** [comment]
Added to Unit test: `https://regex101.com/r/YgW8QO/1`. Unfortunately now I don't see 4 char script, but certainly there were.

**Comment by zloyden** [comment]
This is the current list of domains, so I think we need to expand this expression. I think it's safer to use a modifier `$domain=[...domains]`
```
opalekr.icu,pixsera.net,rtkenale.icu,pikupalate.com,kekolangti.com,imgflyer.xyz,imgez.xyz,imgbigo.xyz,imgopp.xyz,imgex.xyz,imgkoi.xyz,imageking.xyz,imgweng.xyz,cloudgallery.net,imgair.net,imgkaida.xyz,imgfrost.net,imghb.xyz,imgkor.xyz,imgsup.xyz,imgfish.xyz,imgbrd.xyz,imgvivo.xyz,imgtiger.xyz,imgfsh.xyz,imgbird.xyz,imgviv.xyz,imgsupa.xyz,imglion.xyz,imgkr.xyz,imgair.net,imgtigr.xyz,imglin.xyz
```

**Comment by Yuki2718** [comment]
Most of them return 403 error, though might be used again in future. Recently used domain `opalekr.icu,pixsera.net,rtkenale.icu,pikupalate.com,kekolangti.com` are already covered by another regex I added, so the rest are those xyz and `imgair.net` only. How about expanding to cover .net only with domain specified?

**Comment by zloyden** [comment]
> /\.com(\/[a-z]{2,4}){2}\.js$/$script,~third-party,domain=kekolangti.com|pikupalate.com
> /\.icu(\/[a-z]{2,4}){2}\.js$/$script,~third-party,domain=rtkenale.icu|opalekr.icu
> /\.net(\/[a-z]{2,4}){2}\.js$/$script,~third-party,domain=pixsera.net
> The problem was they switch domain too quickly, but now it slowed down.

I guess we can use these rules in this case, but you can adjust them. Throw out first part with `\.com` and add domains to a modifier. If the first part is necessary, use `|` for `.com`, `.icu`, `.net` and you'll get one rule instead of 3.

**Comment by Yuki2718** [comment]
So is it okay just replacing the current filters with `/(?:com|icu|net|xyz)(/[a-z]{2,4}){2}.js$/$script,~third-party,domain=opalekr.icu|pixsera.net|rtkenale.icu|pikupalate.com|kekolangti.com|imgflyer.xyz|imgez.xyz|imgbigo.xyz|imgopp.xyz|imgex.xyz|imgkoi.xyz|imageking.xyz|imgweng.xyz|cloudgallery.net|imgair.net|imgkaida.xyz|imgfrost.net|imghb.xyz|imgkor.xyz|imgsup.xyz|imgfish.xyz|imgbrd.xyz|imgvivo.xyz|imgtiger.xyz|imgfsh.xyz|imgbird.xyz|imgviv.xyz|imgsupa.xyz|imglion.xyz|imgkr.xyz|imgair.net|imgtigr.xyz|imglin.xyz` and closing this PR?

**Comment by zloyden** [comment]
> So is it okay just replacing the current filters with `/(?:com|icu|net|xyz)(/[a-z]{2,4}){2}.js$/$script,~third-party,domain=opalekr.icu|pixsera.net|rtkenale.icu|pikupalate.com|kekolangti.com|imgflyer.xyz|imgez.xyz|imgbigo.xyz|imgopp.xyz|imgex.xyz|imgkoi.xyz|imageking.xyz|imgweng.xyz|cloudgallery.net|imgair.net|imgkaida.xyz|imgfrost.net|imghb.xyz|imgkor.xyz|imgsup.xyz|imgfish.xyz|imgbrd.xyz|imgvivo.xyz|imgtiger.xyz|imgfsh.xyz|imgbird.xyz|imgviv.xyz|imgsupa.xyz|imglion.xyz|imgkr.xyz|imgair.net|imgtigr.xyz|imglin.xyz` and closing this PR?

I think so, but why do you want to close this PR? Just update a link with new Regex for `https://regex101.com/r/eskB1S/1` in the description of the PR, exchange your rule and it can be merged:)
You have a little mistake in the Regex - it should be `(?:com|icu|net|xyz)(\/[a-z]{2,4}){2}.js$` (missed `\`)

**Comment by Yuki2718** [comment]
I expect conflicts, but will try.

**Review by zloyden — APPROVED** [review]

**Pull request merged**